pkg/loghelper/loghelper.go

```diff
@@ -63,3 +66,36 @@ func SetLogFormat() {
 	}
 	logrus.SetFormatter(customFormatter)
 }
+
+func SetHooks() {
+	logrus.AddHook(&writer.Hook{ // Send logs with level higher than warning to stderr
+		Writer: os.Stderr,
+		LogLevels: []logrus.Level{
+			logrus.PanicLevel,
+			logrus.FatalLevel,
+			logrus.ErrorLevel,
+			logrus.WarnLevel,
+		},
+	})
+	logrus.AddHook(&writer.Hook{ // Send info and debug logs to stdout
+		Writer: os.Stdout,
+		LogLevels: []logrus.Level{
+			logrus.InfoLevel,
+			logrus.DebugLevel,
+		},
+	})
+}
+
+// setLogLevel sets the log level for logrus based on the "TNF_LOG_LEVEL" environment variable
+func SetLogLevel() {
+	params := configuration.GetTestParameters()
+
+	var logLevel, err = logrus.ParseLevel(params.LogLevel)
+	if err != nil {
+		logrus.Error("TNF_LOG_LEVEL environment set with an invalid value, defaulting to DEBUG \n Valid values are:  trace, debug, info, warn, error, fatal, panic")
+		logLevel = logrus.DebugLevel
+	}
+
+	logrus.Info("Log level set to: ", logLevel)
```

Comment:
Should be specifier in the format string?

Reply:
Good question - this was in the old code I just moved it to the `loghelper` package.  I'll adjust it.